traveler-ios-ui/TravelerKitUI/ViewControllers/CatalogItemResultViewController.swift

```diff
@@ -36,6 +38,8 @@ class CatalogItemResultViewController: UIViewController {
         let description = catalogItemDetails?.attributedDescription
         description?.setFontFace(font: UIFont.systemFont(ofSize: 17))
         descriptionLabel.attributedText = description
+        wishlistButton.isFilled = catalogItemDetails?.isWishlisted ?? false
+        wishlistButton.tintColor = .red
```

Comment:
any UI attributes you can set in the class or interface builder do so, that way you save the VC from bloated code